### PR TITLE
EN: Episode 3 Story Quest initial draft

### PR DESCRIPTION
**Pull request by ScienceJumps**
One section is the standard menu when a quest's progress will be saved midway, e.g. Ep2's final mission in the Mothership and Ep3's Apprentice mission.

The other is part of a translation for a blue node in Episode 3. It's still under construction as I figure out what they are talking about.

### CODE REVIEW

**Comment by Aida-Enna** [comment]
Looking good! Make sure to replace the JP punctuation with US punctuation though - Like "。" would become "." and the like. \ o / Also, if you translate a file in the /Files/ folder, you need to move it out and into one of the other folders - Only pure JP files can be in /Files/.

Go ahead and make those changes, and the PR will update automatically. Magic!

File: Files/co_021290.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
-text00#0,"""進行状況のセーブ/ロードについて"""
-text01#0,"""このクエストは、進行状況が記録されており<br>クエストを破棄した場合でも、一定の場所から<br>続きを遊ぶことが可能となっています。"""
-text02#0,"""進行状況のセーブについて<br><br>クエストの進行状況がセーブされる際には<br>「クエストの進行を記録しました」というメッセージが表示されます。<br>以後、その場所からクエストを遊ぶことが可能になります。"""
-text03#0,"""進行状況のロードについて<br><br>クエストを受託後、進行状況がセーブされている状態で<br>キャンプシップのテレプールに入ると、確認ウィンドウが表示されます。"""
-text04#0,"""このとき「続きからプレイする」を決定すると<br>前回の進行状況の続きからプレイすることができます。<br>※続きからプレイしても、記録されたデータは削除されません。"""
+text00#0,"""About Saving / Loading mission progress"""
```
Comment (Kyle873) [inline Files/co_021290.csv]: If this line is used in the context of a window title or a sub-section title, it should have title-case capitalization.

**Comment by Aida-Enna** [comment]
You can also remove the `<br>` from the english translations - Those will autowrap and look really really funky if you leave them in :3

File: Files.TOOD/un_034050.csv [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name01#0,"""? ? ?"""
+name02#0,"""? ? ?"""
+un_034050_0010#0,"""<%me>! Over here! I think we lost her!"""
+un_034050_0020#0,"""...Phew. We're safe for now. Hey, Echo, it's..."""
+un_034050_0030#0,"""Ze-Zeno, Zeno! Up, Look up!"""
```
Comment (alama) [inline Files.TOOD/un_034050.csv]: our spell checker script think `Ze` is wrong
You may want to edit `_aspell/PSO2.en_Chat.dict` to add Ze to the word list

Comment (ScienceJumps) [reply Files.TOOD/un_034050.csv]: Do I have permission to do that?

File: Story/Episode 3/un_034050.csv [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+un_034050_0050#0,"""Hm, it's just nice. Doesn't break with a single strike any more. It's quite the upgrade. """
+un_034050_0060#0,"""If you motivate Zieg a little, I guess he can do anything. Don't you agree, Zeno-boy?"""
+un_034050_0070#0,"""Are you trying to kill me? \u3000Maria-san!"""
+un_034050_0080#0,"""It's alright, Labrys' power is lower now. You won't die with a single blow."""
+un_034050_0090#0,"""...Please say you won't kill us before doing that. """
+un_034050_0100#0,"""Speaking of which, Zeno-boy. You seem to quite distracted recently. """
+un_034050_0110#0,"""As your mentor, I felt I needed to motivate you a little as well. """
+un_034050_0120#0,"""Liar! You just wanted to try out the upgraded Labrys!"""
+un_034050_0130#0,"""Oh, very observant. I suppose I don't need to hold back. """
+un_034050_0140#0,"""元から取り繕っても<br>いなかった気がするんだけど……"""
```
Comment (alama) [inline Story/Episode 3/un_034050.csv:16]: Can the rest of `un_034050.csv` be done?

**Comment by Aida-Enna** [comment]
Closed due to inactivity. @ScienceJumps Feel free to comment on here if you're back and wish to pick up where you left off.

**Pull request closed without merging**